drivers/vulkan/vulkan_context.cpp

```diff
@@ -2348,7 +2348,10 @@ Error VulkanContext::swap_buffers() {
 	submit_info.signalSemaphoreCount = 1;
 	submit_info.pSignalSemaphores = &draw_complete_semaphores[frame_index];
 	err = vkQueueSubmit(graphics_queue, 1, &submit_info, fences[frame_index]);
-	ERR_FAIL_COND_V(err, ERR_CANT_CREATE);
+
+	if (err != VK_SUCCESS) {
```

Comment:
The if is redundant, the condition should go in the `ERR_FAIL_COND_V_MSG`. Same below.